src/IB/IBFEInstrumentPanel.cpp

```diff
@@ -338,7 +338,17 @@ IBFEInstrumentPanel::initializeHierarchyIndependentData(IBFEMethod* ib_method_op
     std::vector<std::set<dof_id_type> > temp_node_dof_ID_sets;
     std::vector<std::vector<libMesh::Point> > temp_nodes;
     std::vector<libMesh::Point> meter_centroids;
+    // new API in 1.4.0
```

Comment:
Should we likewise declare our support for libMesh < 1.4 to be deprecated?

Reply:
I think its worth supporting a few old versions of libMesh: we only have checks like this in two places at the moment.

Reply:
i.e., I don't think we should declare it deprecated in some way right now since its very easy to support.